modules/configuring-running-microshift-executable.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-running-microshift-executable"]
+= Configuring and running MicroShift
+:context: creating-microshift-development-environment
+
+Use the following procedure to configure and run a MicroShift deployment 
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+* You have installed the `oc` CLI and `kubectl` utilities. 
+* You have updated CRI-O, networking, and file system customizations. 
+* You have set up your MicroShift development environment. 
+* You have configured and built MicroShift on your virtual machine. 
+* You have prepared the MicroShift executable runtime. 
+
+.Procedure 
+
+. Enter the following commands to initialize MicroShift depdenencies:
++
+[source,terminal]
+----
+$ sudo systemctl enable crio
+$ sudo systemctl start microshift
+----
++
+[NOTE]
+====
+The CRI-O service is started automatically by systemd as one of the MicroShift dependencies. You must enable the CRO-O service to allow MicroShift to run as a standalone executable. 
+====
+
+. Wait until all pods are up and running. To see the list of pods, enter the following command:
++
+[source,terminal]
+----
+$ watch sudo $(which oc) --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig get pods -A
+----
+
+. Stop the MicroShift service and clean up the images and configuration data:
++
+[source,terminal]
+----
+$ echo 1 | ~/microshift/hack/cleanup.sh
+----
+
+. Run the MicroShift executable file by entering the following command:
++
+[source,terminal]
+----
+$ nohup sudo ~/microshift/_output/bin/microshift run >> /tmp/microshift.log &
+----
+
+. Examine the `/tmp/microshift.log` file to ensure that MicroShift successfully started. 
+
+. Copy the `kubeconfig` file to the default location that can be accessed without administrative privileges by running the following commands:
++
+[source,terminal]
+----
+$ mkdir -p ~/.kube/
+$ sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
+----
+
+. You can verify that MicroShift is running by entering the following commands:
++
+[source,terminal]
+----
+$ oc get cs 
+$ watch oc get pods -A 
+----
```

Comment:
```suggestion
```

We would only want developers to follow these steps, so let's not put them in the user docs.